src/main/scala/edu/ie3/simona/agent/grid/congestion/VoltageRange.scala

```diff
@@ -208,17 +211,17 @@ object VoltageRange {
 
     // build initial range
     val range = VoltageRange(
-      voltageLimits.vMax.subtract(maxVoltage._2),
-      voltageLimits.vMin.subtract(minVoltage._2),
+      voltageLimits.vMax.subtract(maxVoltage._2).toSquants,
+      voltageLimits.vMin.subtract(minVoltage._2).toSquants,
     )
 
-    // updating the voltage range prevent or cure line congestions
-    val deltaV = calculatePossibleVoltageDeltaForLines(
+    // updating the voltage range prevent or heal line congestions
+    val deltaV = calculateVoltageDeltaFromLineCurrent(
       nodeResMap,
```

Comment:
Do you see a chance and sense to convert the nodeResMap to squants somewhere early? If so, one could kick the javax Dimensionless out here.